packages/dev/src/quickstarts-data/mas-guides/alert-note-prereq/README.adoc

```diff
@@ -0,0 +1,40 @@
+[id="chap-alert-note-prereq"]
+= Alert, note, and prerequisites example quickstart
+
+.Prerequisites
+* You want to see some new features condensed into one example quickstart.
+
+ifdef::qs[]
+[#description]
+Preview newly added features: Alert, note, and prerequisite sections rendered using Patternfly React Components.
+
+[#introduction]
+Welcome to this example quickstart making it easy to see some new features, which will be included in a design rework of the quickstart drawer.
+endif::[]
+
+[id="task-1_{context}",module-type="proc"]
+== Look at the new features
+
+In this Quick Start you will preview some design changes.
+
+.Prerequisites
+* Take a look at the new design changes, including this prerequisites section.
+* Also take a look at the note and alert sections.
+
+.Procedure
+. Note section
++
+NOTE: I am a Note rendered by Patternfly React Components.
```

Comment:
Is the parser simply triggering on uppercase `NOTE:` and `IMPORTANT:` keywords? If that's the case that might not be specific enough, for example users could use this text without wanting to invoke the associated PF component

Reply:
These are keywords in Asciidoc with syntax highlighting and special markdown rendering.